common/group.ts

```diff
@@ -1,8 +1,11 @@
+import { JSONContent } from '@tiptap/core'
+
 export type Group = {
   id: string
   slug: string
   name: string
   about: string
+  dashboard: JSONContent | null
```

Comment:
Just a note that you'd also have to run a script to update existing groups to have `dashboard: null` because otherwise the type would be inaccurate since they are undefined.

An alternate way to do this would be to give dashboard an optional type:
```dashboard?: JSONContent```
That seems a bit simpler to me and more consistent with how we typically add fields. To delete the dashboard field, you'd have to use the special `FieldValue.delete()` value from firestore.

A third option is to have both optional and null in the type, but that seems less clean than the second option.